### DISCUSSION TITLE
Workflow trigger workflow

### DISCUSSION THREAD

**Discussion by Smile-Autra** [PrefectHQ/prefect#5593] [Q&A] [answered]
In a real industrial scenario, there are often many ETL tasks. So does `Prefect` support trigger a flow by another flow, and we can set and view their dependency on the web UI?

**Comment by zanieb** [comment]
Yes a workflow can be triggered by another workflow.

v1: via `create_flow_run` task https://docs.prefect.io/orchestration/flow-runs/creation.html#task
v2: via call https://orion-docs.prefect.io/concepts/flows/#subflows

You can view the dependency in the UI as well

v1: via artifacts https://docs.prefect.io/orchestration/concepts/artifacts.html#creating-link-artifacts
v2: built-in to the graph view

You cannot "set" dependencies via the UI, but for both versions you can set a description / readme for the flow that includes references to the other flows.

**Comment by Smile-Autra** [comment, marked as answer]
Thanks
